# Generated by Django 2.2.16 on 2022-11-10 13:05

from django.db import migrations, models
import users.validators


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='role',
            field=models.CharField(choices=[('user', 'Пользователь'), ('admin', 'Администратор'), ('moderator', 'Модератор')], default='user', max_length=10, verbose_name='Роль'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'Пользователь с таким именем пользователя уже существует.'}, help_text='Обязательное поле. Не более 150 символов.Только буквы, цифры и символы @/./+/-/_.', max_length=150, unique=True, validators=[users.validators.CustomUsernameValidator()], verbose_name='username'),
        ),
    ]
